doc/resync_broken_replica.md

```diff
@@ -0,0 +1,18 @@
+# Re-sync a broken replica
+
+Due to unforeseen circumstances, if you end up with an empty postgres data directory on one of your replicas but your _master_ is intact, you will receive a message that looks something like this:
+
+```
+different local dbUID but init mode is none, this shouldn't happen. Something bad happened to the keeper data. Check that keeper data is on a persistent volume and that the keeper state files weren't removed
+```
+
+This is because all keepers _require_ persistent storage and will not auto "re-sync" just because they find an empty data directory while still having an entry in the clusterdata.
```

Comment:
I'll add a link to the `architecture.md` doc that clearly explains this.

Reply:
@sgotti I've added the link under the Misc. section, and also updated `architecture.md` to specify the behaviour in case of a cluster ID mismatch. Please review.